v5/pivotal/stories.go

```diff
@@ -211,17 +213,66 @@ func (service *StoryService) List(projectID int, filter string) ([]*Story, error
 	return stories, nil
 }
 
-func newStoriesRequestFunc(client *Client, projectID int, filter string) func() *http.Request {
+func newStoriesRequestFunc(client *Client, projectID int, filter string, fields []string) func() *http.Request {
 	return func() *http.Request {
 		u := fmt.Sprintf("projects/%v/stories", projectID)
 		if filter != "" {
 			u += "?filter=" + url.QueryEscape(filter)
+			if len(fields) == 0 {
+				fields = DefaultFields
+			}
```

Comment:
This seems redundant given that `fieldsToQuery` also sets the default fields.

I do wonder if not including the param would be less surprising if there are no fields passed, but I am fine with it as is given that we are only using this internally.

Reply:
Yes, I agree. Thank you, I have removed it. 👍